pkg/template/template.go

```diff
@@ -94,3 +108,36 @@ func mapType(t Type) prometheus.ValueType {
 		return prometheus.UntypedValue
 	}
 }
+
+func (m *MetricTemplate) sendMergedMetric(vals []float64, labels [][]string, ch chan<- prometheus.Metric) {
+	l := make(map[string]string)
+	l["customizations"] = strings.Replace(m.Name, fmt.Sprintf("%s_", metricShootsCustomPrefix), "", 1)
+	var noLabels = len(labels) == 0
+
+	if noLabels {
+		// TODO  closer look
+		if len(vals) == 0 {
+			return
+		}
+		var desc = prometheus.NewDesc(fmt.Sprintf("%s_merged", metricShootsCustomPrefix), "Collection of all collected customization metrics.", nil, l)
```

Comment:
I think we can drop the `_merged` suffix. Is there any reason that we need it?

Reply:
I added the `_merged` suffix to highlight the difference between the specific version e.g.`garden_shoots_custom_addon_kubedashboard_total` and the generic `garden_shoots_custom`. But sure will remove it.

Reply:
https://github.com/gardener/gardener-metrics-exporter/pull/27/commits/216ab326350b8c200516a548e9e7d61416f87898